src/modules/Dropdown/Dropdown.story.mdx

```diff
@@ -0,0 +1,139 @@
+import { Meta, Story, Preview } from "@storybook/addon-docs/blocks";
+import LinkTo from "@storybook/addon-links/react";
+import { DropdownStory, MultipleDropdownStory } from "./Dropdown.story.tsx";
+
+<Meta title="Components | Dropdown" />
+
+# Dropdown
+
+This component can be used to select a value from a list. It uses the `Popover` component underneath to show the options.
+
+<Preview>
+  <Story name="Dropdown">
+    <DropdownStory />
+  </Story>
+</Preview>
+
+<Preview>
+  <Story name="Multiple Dropdown">
+    <MultipleDropdownStory />
+  </Story>
+</Preview>
+
+### Available props
+
+All the props are passed through to the `Popover` element, so refer to its documentation for more customization options.
+
+<table width="100%">
+  <thead>
+    <tr>
+      <th>Prop Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>items</td>
+      <th>{"IItem: { id: string, text: string }"}</th>
+      <td>
+        The list of options to be presented. This is a generic type, so it may
+        contain additional attributes.
+      </td>
+    </tr>
+    <tr>
+      <td>onItemSelected</td>
+      <th>(item) => void</th>
+      <td>Called when an item is selected.</td>
+    </tr>
+    <tr>
+      <td>selectedItem</td>
+      <th>IItem (optional)</th>
```

Comment:
I placed my `(optional)`s on the prop column 🙈should we decide what looks/is better? 😛